src/react-components/room/ReactionButtonContainer.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState, useCallback } from "react";
+import PropTypes from "prop-types";
+import { ReactComponent as ReactionIcon } from "../icons/Reaction.svg";
+import { ToolbarButton } from "../input/ToolbarButton";
+
+export function ReactionButtonContainer({ scene }) {
```

Comment:
We'll need to do some thinking around the UX for "send to 3D world" vs "send to chat" vs "send to both" . It is surprising to me that there's a "react" (emoji) menu in the toolbar rather than in the chat sidebar or the "Place" popover menu